src/sys/windows/mod.rs

```diff
@@ -1,3 +1,19 @@
+/// Helper macro to execute a WinSock system call that returns an `io::Result`.
+#[allow(unused_macros)]
+macro_rules! wsa_syscall {
+    ($fn: ident ( $($arg: expr),* $(,)* ), $err_value: expr) => {{
+        let res = unsafe { windows_sys::Win32::Networking::WinSock::$fn($($arg, )*) };
+        if PartialEq::eq(&res, &$err_value) {
+            Err(io::Error::from_raw_os_error(unsafe {
```

Comment:
Seems unlikely, but just in case `std::io` isn't imported.
```suggestion
            Err(std::io::Error::from_raw_os_error(unsafe {
```